### PR TITLE
enhancement: Update bashisms.yml

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [x] Hotfix

## Description
- Changed from `apt` to `apt-get`.
- Added `apt-get update` prior to package installation, in accordance with the documentation at [GitHub Docs](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners).
- Use xargs to run checks a faster by parallel processing.

## Testing
- Tested on my repo works fine.

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: .github/workflows/bashisms.yml [hunk 1]
```diff
@@ -19,9 +19,8 @@ jobs:
       - uses: actions/checkout@v2
       
       - name: Install devscripts
-        run: sudo apt install devscripts
+        run: sudo apt-get update && sudo apt-get install devscripts
```
Comment (adamperkowski) [inline .github/workflows/bashisms.yml:22]: No need to update here. Containers provided by GitHub are already pre-updated.
```suggestion
        run: sudo apt install devscripts
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

File: .github/workflows/bashisms.yml [hunk 2]
```diff
@@ -19,9 +19,8 @@ jobs:
...
 
-      - name: Concatenate all .sh files and check for bashisms
+      - name: Check for bashisms
         working-directory: tabs
-        run: |
-          find . -name '*.sh' -exec checkbashisms {} + > all_scripts.sh
+        run: find . -name '*.sh' | xargs -P 4 -n 1 checkbashisms
```
Comment (adamperkowski) [inline .github/workflows/bashisms.yml:26]: This is a good change actually. I wanted "fix" this in my PR when I first implemented this CI, but I forgot and it got merged.

File: .github/workflows/bashisms.yml (hunk 1 above)
Comment (jeevithakannan2) [reply .github/workflows/bashisms.yml:22]: Please check with https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners

**Review by adamperkowski — APPROVED** [review]

File: .github/workflows/bashisms.yml (hunk 1 above)
Comment (jeevithakannan2) [reply .github/workflows/bashisms.yml:22]: No problem thank you for taking a look

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**